admin/test_langs.php

```diff
@@ -152,7 +152,7 @@ function get_lang_files( $p_path ) {
  * @return void
  */
 function checklangdir( $p_path ) {
-	$t_path = rtrim( $p_path, DIRECTORY_SEPARATOR ) . DIRECTORY_SEPARATOR . 'lang' . DIRECTORY_SEPARATOR;
+	$t_path = rtrim( $p_path, DIRECTORY_SEPARATOR ) . '/lang/';
```

Comment:
You still have directory separator used. The trim and concat should use the same construct.

Reply:
ditto.